data/hai/hai missions.txt

```diff
@@ -1932,75 +1932,101 @@ mission "Station Exploration Abandon Ship"
 	destination
 		attributes "unfettered"
 	passengers 1
 	to offer
-		random < 30
-		has "outfit: Quantum Keystone"
+		random < 10
+		or
+			has "outfit: Quantum Keystone"
+			has "outfit: Quantum Key Stone"
+			has "outfit: Shield Beetle Pendant"
+			has "outfit: Tree Skeleton Key Stone"
 		has "First Contact: Hai: offered"
 	on offer
 		conversation
-			`It is an unusually foggy day on <source>. The air is so thick with moisture that you cannot see more than a few paces in front of you. Perhaps that is why you are so startled when you turn around and see a Hai standing right next to you. He is so close you can see dewdrops on his whiskers, and you immediately notice the prominent jewelry that he is wearing. He appears to be quite old, judging by his stooped-over posture and the white fur around his muzzle.`
-			`	"I see that you are wiser than most humans here," he says, pointing towards your ship. "You carry a Keystone, a shield against the tempest of space. Let no one else tell you otherwise. Are you willing and able to take me to <destination>? I can pay <payment>."`
+			`It is an unusually foggy day on <origin>; the air is so thick with moisture that you cannot see more than a few paces in front of you. Perhaps that's why you're so startled when you turn around and see a Hai standing right next to you. He is so close you can see dewdrops on his whiskers, and you immediately notice the prominent jewelry that he is wearing. He appears to be quite old, judging by his stooped-over posture and the white fur around his muzzle.`
+			branch pendant
+				has "outfit: Shield Beetle Pendant"
+			branch normal
+				has "outfit: Quantum Keystone"
+			branch remnant
+				has "outfit: Quantum Key Stone"
+			branch huh
+				has "outfit: Tree Skeleton Key Stone"
+			label normal
+			`	"I see that you are wiser than most humans here," he says, pointing towards your ship. Its hatch is open, prominently displaying your Keystone to any potential travellers. "You carry a shield against the tempest of space. Let no one else tell you otherwise." He stops to catch his breath. "Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
+				goto choice
+			label remnant
+			`	"The keystone you carry is a strange one, human," he says, pointing towards your ship. Its hatch is open, prominently displaying your Keystone to any potential travellers. He walks up to it to get a closer look. "Very unusual. No proper mounting either - must be a human design." He sighs. "Well, at least the Keystone itself looks authentic. It will have to do." He turns to you. "You carry a shield against the tempest of space. Let no one else tell you otherwise. If your people have access to Keystones as well, then I hope they too become wise to its power, and learn to treat them with more reverence. Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
```

Comment:
Dunno about the capitalization of 'keystone' here. I'll have to check.

Reply:
It's mixed. The description of the outfit uses lowercase `keystone` (second paragraph), and I assume that's most relevant here, since he's not referring to the item in the Outfitter, but to the keystone itself.

<img width="281" alt="image" src="https://github.com/endless-sky/endless-sky/assets/98854112/21c52aff-31fb-4e66-8c20-ac0ff84f59c6">

Reply:
#8355 quantum keystone is not capitalized, Quantum Key Stone is.

Reply:
Tibe mentioned that 'keystone' isn't capitalized outside of HR. Since 'keystone' seems to be the standard for most of the game, I think it should be lowercase along with 'key stone' for the same reason.